modules/rocksdb-common/src/main/java/org/apache/ignite/internal/rocksdb/ColumnFamily.java

```diff
@@ -69,7 +76,7 @@ private ColumnFamily(RocksDB db, ColumnFamilyHandle handle) throws RocksDBExcept
     public static ColumnFamily create(RocksDB db, ColumnFamilyDescriptor descriptor) throws RocksDBException {
```

Comment:
How about changing method name to highlight that the result is going to own the column options? But I cannot think of any good name yet... `createOwningOptions` seems a bit too clumsy and specific